tests/api_resources/test_topup.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, division, print_function
+
+import stripe
+from tests.helper import StripeTestCase
+
+
+TEST_RESOURCE_ID = 'tu_123'
+
+
```

Comment:
Nit: Unnecessary extra lines?

Reply:
(Similarly to the comment above, I believe PEP 8 will require this doubled up blank line too.)